test/model/directus_user_test.dart

```diff
@@ -36,5 +36,33 @@ main() {
       expect(sut.firstname, null);
       expect(sut.lastname, null);
     });
+
+    test('Reading first, last and full name', () {
```

Comment:
Could you add tests for edge cases (firstname and/or lastname are null for example) ? Thanks!